shopping_mall/backend/app/services/order_processor.py

```
@@ -0,0 +1,251 @@
+            db.query(ShopTicket)
+            .filter(
+                ShopTicket.order_id == order.id,
+                ShopTicket.action_type == "cancel",
+                ShopTicket.status == "received",
+            )
+            .first()
+        )
+        if existing:
+            raise ValueError(
+                f"이미 처리 대기 중인 취소 티켓이 있습니다: ticket_id={existing.id}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)